python/lsst/sims/GalSimInterface/galSimInterpreter.py

```diff
@@ -509,6 +520,67 @@ def writeImages(self, nameRoot=None):
 
         return namesWritten
 
+    def open_centroid_file(self, detector_name):
```

Comment:
Given the code in `_writeObjectToCentroidFile`, it would be clearer to change the argument name from `detector_name` to `centroid_name` here and below.